content/posts/project-structure-on-gcp/project-structure-on-gcp.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Project Structure on GCP"
+description: "A short explanation of projects in NAV's GCP setup"
+date: 2020-11-30T14:51:27+01:00
+draft: true
+author: Gøran Berntsen
+tags: [gcp, nais]
+---
+
+In terms of the [GCP resource hierarchy](https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy), NAV is an **_Organization_**, each team is assigned two **_projects_** (one for dev and one for prod), and the teams can provision **_resources_** within these projects. In theory, these resources can be any of the [services offered by Google Cloud Platform](https://cloud.google.com/docs/overview/cloud-platform-services). In practice, some restrictions apply. More on this later.
+
+The team projects are automatically set up when a team is added to [teams.yaml](https://github.com/navikt/teams). These projects will be named *{teamname}-dev* and *{teamname}-prod*, and owner/administrator roles will be assigned based on the associated Azure AD group. In addition, the team is assigned its own namespace in the nais kubernetes clusters. The namespace is named the same as the team in teams.yaml. 
```

Comment:
The teams repo is private, include a note about this so that people don't think it's a broken link?